src/content/reference/react/useDeferredValue.md

```diff
@@ -16,11 +16,11 @@ const deferredValue = useDeferredValue(value)
 
 ---
 
-## Reference {/*reference*/}
+## Referensi {/*reference*/}
 
 ### `useDeferredValue(value)` {/*usedeferredvalue*/}
 
-Call `useDeferredValue` at the top level of your component to get a deferred version of that value.
+Panggil fungsi `useDeferredValue` di tingkat atas komponen Anda untuk mendapatkan versi yang ditangguhkan dari nilai tersebut.
```

Comment:
Ada dua terjemahan berbeda untuk defer: tangguh dan tunda.
Mau pakai yang mana? Coba check ada gak di Glossary.
Yang penting konsisten sih

Reply:
Thanks for the review, mas @mazipan 


on this [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium), there's no word `defer`. so for keep consistency, I'll choose **"tangguh"** instead of **"tunda"**.

changes delivered on this commit https://github.com/reactjs/id.react.dev/pull/360/commits/56be44dbcf41115273546d5517abe4670bb809c4

Reply:
cc: @zainfathoni @mhaidarhanif @r17x @resir014